packages/synapse-react-client/src/components/IconList.tsx

```diff
@@ -29,6 +31,15 @@ function IconList(props: IconListProps) {
   let noMatch: boolean = false
   const css = useTheme ? 'icon-list themed' : 'icon-list'
   const componentCss = useBackground ? `${css} bg-circle` : css
+  const queryContext = useQueryContext()
+  const { addValueToSelectedFacet } = queryContext
+
+  const handleIconClick = (dataType: 'string') => {
+    const facet: UniqueFacetIdentifier = {
+      columnName: 'dataType',
+    }
+    addValueToSelectedFacet(facet, dataType)
```

Comment:
I verified that addValueToSelectedFacet in TableQueryReducerActions handles the case when the caller attempts to add the same facet value more than once (duplicate value is not pushed).  So this should be fine!